views/movies/new.hbs

```diff
@@ -0,0 +1,21 @@
+<h2>Add New Movie</h2>
+
+  <form action="/movies/new" method="GET">
+    <label for="title">Title</label> 
+    <input id="title" type="text" name="title">
+    
+    <label for="genre">Genre</label> 
+    <input id="genre" type="text" name="genre">
+    
+    <label for="plot">Plot</label> 
+    <input id="plot" type="text" name="plot">
+
+    <label for="cast">Cast</label> 
+    <select id="cast" name="cast" size="3" multiple>
+    <option value="Giulia Baile">Giulia Baile</option>
+    <option value="Winter">Winter</option>
+    <option value="Natalie Dupont">Natalie Dupont</option>
```

Comment:
This will not change if we add other celebrities to the database. It is better for this to be dynamic depending on the celebrities collection. See my comment above.

The value for each option should be the ID of the celebrity since this is what should be saved in the database.